features/ubuntu_pro.feature

```diff
@@ -613,3 +613,300 @@ Feature: Command behaviour when auto-attached in an ubuntu PRO image
            | xenial  | n/a      | disabled | disabled | libkrad0  | jq       | n/a       | cis        |
            | bionic  | disabled | disabled | disabled | libkrad0  | bundler  | n/a       | cis        |
            | focal   | n/a      | n/a      | disabled | hello     | ant      | enabled   | usg        |
+
+    @slow
```

Comment:
Now that this is moved to the other file and without the overrides, this part can be removed, right?